src/modules/launcher/PowerLauncher/ViewModel/ResultsViewModel.cs

```diff
@@ -217,25 +216,9 @@ public bool IsContextMenuItemSelected()
         /// To avoid deadlock, this method should not called from main thread
```

Comment:
Is the <summary> comment still accurate?

Reply:
No. I will remove this. Thanks for catching this.